external_search.py

```diff
@@ -1655,6 +1715,7 @@ def from_worklist(cls, _db, worklist, facets):
         fiction = inherit_one('fiction')
         audiences = inherit_one('audiences')
         target_age = inherit_one('target_age')
+        collections = inherit_one('collection_ids') or library
```

Comment:
Changes in this method are necessary to support synthetic lanes that have no associated Library -- crawlable lanes, basically -- without special code.